src/components/refactor/dialogs/lines-attach-to-split-lines/lines-attach-to-split-lines-dialog.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useSnackMessage } from '@gridsuite/commons-ui';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { sanitizeString } from 'components/dialogs/dialogUtils';
+import PropTypes from 'prop-types';
+import {
+    ATTACHED_LINE_ID,
+    BUS_BAR_SECTION_ID,
+    LINE_TO_ATTACH_TO_ID_1,
+    LINE_TO_ATTACH_TO_ID_2,
+    REMPLACING_LINE_ID_1,
+    REMPLACING_LINE_ID_2,
+    REMPLACING_LINE_NAME_1,
+    REMPLACING_LINE_NAME_2,
+    VOLTAGE_LEVEL_ID,
+} from 'components/refactor/utils/field-constants';
+import yup from 'components/refactor/utils/yup-config';
+import React, { useCallback, useEffect } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { linesAttachToSplitLines } from 'utils/rest-api';
+import ModificationDialog from 'components/refactor/dialogs/commons/modificationDialog';
+
+import LinesAttachToSplitLinesForm from './lines-attach-to-split-lines-form';
+
+/**
+ * Dialog to create a load in the network
+ * @param studyUuid the study we are currently working on
+ * @param currentNode The node we are currently working on
+ * @param editData the data to edit
+ * @param dialogProps props that are forwarded to the generic ModificationDialog component
+ */
+
+const emptyFormData = {
+    [LINE_TO_ATTACH_TO_ID_1]: '',
+    [LINE_TO_ATTACH_TO_ID_2]: '',
+    [ATTACHED_LINE_ID]: '',
+    [VOLTAGE_LEVEL_ID]: '',
+    [BUS_BAR_SECTION_ID]: '',
+    [REMPLACING_LINE_ID_1]: '',
+    [REMPLACING_LINE_NAME_1]: '',
+    [REMPLACING_LINE_ID_2]: '',
+    [REMPLACING_LINE_NAME_2]: '',
```

Comment:
use null default value when dealing with autocomplete/select inputs
add nullable() to those fields in yup

currently, there are wrong error messages when clearing those fields